src/apps/subgraph_data/treasury_assets.py

```diff
@@ -0,0 +1,82 @@
+from src.apps.subgraph_data.protocol_metrics import \
+    get_df_for_treasury_asset_by_address
+
+from src.apps.util.constants import BCT_ERC20_CONTRACT, BCT_USDC_PAIR, \
+     KLIMA_BCT_PAIR, KLIMA_MCO2_PAIR, KLIMA_USDC_PAIR, \
+     MCO2_ERC20_CONTRACT, NBO_ERC20_CONTRACT, \
+     NCT_ERC20_CONTRACT, UBO_ERC20_CONTRACT, USDC_ERC20_CONTRACT
+
+import plotly.graph_objects as go
+
+
+MODE_ASSET_VALUE_IN_USD = "mode_usd"
+MODE_CARBON_CUSTODIED = "mode_carbon_custodied"
+
+
+class TreasuryAssetWrapper:
+    def __init__(self, mode):
+        self.mode = mode
+        treasuryAssetData = []
+        treasuryAssetData.append(SingleAssetData(
+            name="BCT", address=BCT_ERC20_CONTRACT, color="#eff542", custodiesCarbon=True))
+        treasuryAssetData.append(SingleAssetData(
+            "KLIMA BCT LP", KLIMA_BCT_PAIR, "#f1c232", True))
+        treasuryAssetData.append(SingleAssetData(
+            "BCT USDC LP", BCT_USDC_PAIR, "#bf9000", False))
+        treasuryAssetData.append(SingleAssetData(
+            "MCO2", MCO2_ERC20_CONTRACT, "#93c47d", True))
+        treasuryAssetData.append(SingleAssetData(
+            "KLIMA MCO2 LP", KLIMA_MCO2_PAIR, "#6aa84f", True))
```

Comment:
Soo technically the MCO2 in KLIMA/MCO2 LP shouldn't be considered "carbon custodied" based on the current protocol mechanics because we don't defend any particular MCO2 price in KLIMA terms (unlike BCT where the protocol will defend 1 BCT backing)